packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -139,6 +139,21 @@ const propTypes = {
    * Boolean indicating whether or not the DataGrid should hide the column headers.
    */
   hasVisibleColumnHeaders: PropTypes.bool,
+
+  /**
+   * @private
+   * A Unique Identifier of the [column](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#column).
+   * If provided, column with specified identifier will be highlighted in data-grid.
+   *
+   * ![IMPORTANT](https://badgen.net/badge/UX/Design-Standards/blue) The column highlight feature should be limited specifically to
+   * time and timeline concepts only, best used with special instruction and guidance from User Experience to ensure proper standards.
+   */
+  columnHighlightId: PropTypes.string,
+
+  /**
+   * The color to be used for highlighting a column.
+   */
+  colorHighlightColor: PropTypes.oneOf(['orange', 'green']),
```

Comment:
Why only one of them is private? Shouldn't both be private or both visible on doc site?

Reply:
Yes both should be private for DataGrid.  Both will be visible only under flowsheet props table.